### ISSUE TITLE
minikube 0.28 not boot with kubernetes 1.11.0

### ISSUE THREAD

**Issue by remoe** [kubernetes/minikube#2966]
**BUG REPORT**

**Environment**:

**Minikube version** (use `minikube version`): 0.28
- **OS** (e.g. from /etc/os-release): Windows
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): hyperv
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`):
- **Install tools**:
- **Others**:

**What happened**:
minikube startup

**What you expected to happen**:
It fails on startup

**How to reproduce it** (as minimally and precisely as possible):

minikube start --vm-driver=hyperv --kubernetes-version="v1.11.0" --hyperv-virtual-switch="Default Switch" --memory=4096 --alsologtostderr -v 5

**Output of `minikube logs` (if applicable)**:
```
Jul 08 13:39:41 minikube kubelet[3263]: E0708 13:39:41.985005    3263 reflector.go:205] k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:47: Failed to list *v1.Pod: Get https://172.31.45.76:8443/api/v1/pods?fieldSelector=spec.nodeName%3Dminikube&limit=500&resourceVersion=0: dial tcp 172.31.45.76:8443: connect: connection refused
```

**Anything else do we need to know**:
The api-server does not start because of this error:

```error: admission-control and enable-admission-plugins/disable-admission-plugins flags are mutually exclusive```

**Comment by remoe** [comment]
Is it solved by: https://github.com/kubernetes/minikube/commit/81440feabb5f4fb6d04fa04f6901c54e6602ed50

?

**Comment by aojea** [comment]
I tried minikube with that patch in linux and it solved this issue.

Just for reference for other users, you can obtain minikube with that patch from this links:

https://storage.googleapis.com/minikube-builds/2943/minikube-darwin-amd64
https://storage.googleapis.com/minikube-builds/2943/minikube-linux-amd64
https://storage.googleapis.com/minikube-builds/2943/minikube-windows-amd64.exe

**Comment by dlorenc** [comment]
Fixed in 0.28.11. #2943
